### ISSUE TITLE
data-graph field in RACK cli .yaml wasn't applied

### ISSUE THREAD

**Issue by cuddihyge** [ge-high-assurance/RACK#434]
My YAML contains data-graph of http://paultest/data

![image](https://user-images.githubusercontent.com/19863188/120675785-63cb3400-c463-11eb-953c-50eb667c4936.png)

But the ingestion goes to http://rack001/data

![image](https://user-images.githubusercontent.com/19863188/120675942-865d4d00-c463-11eb-87d5-9efe4755961b.png)

@glguy Let me know if this is something obvious/easy on your end, or whether it seems to be happening in semtk3_python.  I'm happy to pair.

**Comment by Ptival** [comment]
Possibly related to ade89f0aa02f85f12f844e16e01d64024821bca7 ?

**Comment by glguy** [comment]
Yeah, this was a bug that we fixed like @Ptival mentions. until then you get the new version you'll need to use the `--data-graph` flag when importing the data to redirect it out of `http://rack001/data`
